docs/config-ha.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "Configuring high-availability components"
+linkTitle: "Configuring high-availability components"
+weight: 20
+type: "docs"
+---
+
+Active/passive high availability (HA) is a standard feature of Kubernetes APIs that helps to ensure that APIs stay operational if a disruption occurs. In an HA deployment, if an active controller crashes or is deleted, another controller is available to take over processing of the APIs that were being serviced by the controller that is now unavailable.
+
+Active/passive HA in Knative is available through leader election, which can be enabled after Knative Serving control plane is installed.
+
+When using a leader election HA pattern, instances of controllers are already scheduled and running inside the cluster before they are required. These controller instances compete to use a shared resource, known as the leader election lock. The instance of the controller that has access to the leader election lock resource at any given time is referred to as the leader.
+
+HA functionality is available on Knative for the `autoscaler-hpa`, `controller` and `activator` components.
+
+## Enabling leader election
```

Comment:
I feel like we should clearly indicate a feature-state before asking folks to enable things.

Given that this is still opt-in and baking, and doesn't yet cover everything, I'd probably call this "alpha" still (cc @pmorie for thoughts, and since we haven't discussed a designation).

Personally, I'd love to more towards having this enabled everywhere by default and let these docs focus more on how to tune things (e.g. lease refreshes, etc).

Reply:
I would be okay calling it 'alpha' in this PR, but we'll need to discuss a scheme for these things (hopefully we can just use alpha, beta, GA)